book/src/ch1/the_display_control.md

```diff
@@ -0,0 +1,105 @@
+you might see it under that name if you read other guides.
+
+Among IO Registers, it's one of the simpler ones, but it's got enough complexity
+that we can get a hint of what's to come.
+
+Also it's the one that you basically always need to set at least once in every
+GBA game, so it's a good starting one to go over for that reason too.
+
+The Display Control is a `u16` value located at `0x0400_0000`.
+
+## Video Modes
+
+The lowest three bits (0-2) let you select from among the GBA's six video modes.
+You'll notice that 3 bits allows for eight modes, but the values 6 and 7 are
+prohibited.
+
+Modes 0, 1, and 2 are "Tiled" modes. These are actually the modes that you
+should eventually learn to use as much as possible. It lets the GBA's limited
+video hardware do as much of the work as possible, leaving more of your CPU time
+for gameplay computations. However, they're also complex enough to deserve their
+own demos and chapters later on, so that's all we'll say about them for now.
+
+Modes 3, 4, and 5 are "Bitmap" modes. These let you write individual pixels to
+locations on the screen.
+
+* **Mode 3** is full resolution (240w x 160h) RBG15 color. You might not be used
+  to RGB15, since modern computers have 24 or 32 bit colors. In RGB15, there's 5
+  bits for each color channel stored within a `u16` value, and the highest bit is
+  simply ignored.
+* **Mode 4** is full resolution paletted color. Instead of being a `u16` color, each
+  pixel value is a `u8` palette index entry, and then the display uses the
+  palette memory (which we'll talk about later) to store the actual color data.
+  Since each pixel is half sized, we can fit twice as many. This lets us have
+  two "pages". At any given moment only one page is active, and you can draw to
+  the other page without the user noticing. You set which page to show with
+  another bit we'll get to in a moment.
+* **Mode 5** is full color, but also with pages. This means that we must have a
+  reduced resolution to compensate (video memory is only so big!). The screen is
+  effectively only 160w x 128h in this mode.
+
+## CGB Mode
+
+Bit 3 is effectively read only. Technically it can be flipped using a BIOS call,
+but when you write to the display control normally it won't write to this bit,
+so we'll call it effectively read only.
+
+This bit is on if the CPU is in CGB mode.
+
+## Page Flipping
+
+Bit 4 lets you pick which page to use. This is only relevent in video modes 4 or
+5, and is just ignored otherwise. It's very easy to remember: when the bit is 0
+the 0th page is used, and when the bit is 1 the 1st page is used.
+
+The second page always starts at `0x0600_A000`.
+
+## OAM, VRAM, and Blanking
+
+Bit 5 lets you access OAM during HBlank if enabled. This is cool, but it reduces
+the maximum sprites per scanline, so it's not default.
+
+Bit 6 lets you adjust if the GBA should treat Object Character VRAM as being 2d
```

Comment:
This needs a better explanation, because neither Tonc nor GBATEK do a good job of it (though Tonc at least has a [visual aid](http://www.coranac.com/tonc/text/regobj.htm#ssec-map)). 1D mode is self-explanatory; 2D mode treats VRAM as a canvas 32 tiles wide, such that the sprites remain clear when you map out the whole of VRAM.

Reply:
I added a note at the top that we don't expect the reader to understand a lot of this right away, and I filed an issue for it so we don't forget.